server/vitals.js

```diff
@@ -140,8 +140,32 @@ async function checkButtonBatteries() {
   }
 }
 
+function logClientMessage(clientMessages) {
+  if (Object.keys(clientMessages).length > 0) {
+    Object.values(clientMessages).forEach(({ client, disconnectedButtons, reconnectedButtons }) => {
+      const clientDisplayName = client.displayName
+      let buttonLogMessage = ''
+
+      if (disconnectedButtons.length > 0) {
+        const buttonNames = disconnectedButtons.join(', ')
+        buttonLogMessage += ` The following buttons have been disconnected: ${buttonNames}.`
+      }
+
+      if (reconnectedButtons.length > 0) {
+        const buttonNames = reconnectedButtons.join(', ')
+        buttonLogMessage += ` The following buttons have been reconnected: ${buttonNames}.`
+      }
+
+      helpers.logSentry(`Button status change for: ${clientDisplayName}.${buttonLogMessage}`)
```

Comment:
This looks good to me. 

Will these button names be in any particular order?

Reply:
Thanks @trisapeace! The button order will be based on the `buttonVitals` array, which comes from a db query that orders the buttons based on the `created_at` date in the `buttons_vitals_cache` table.

Reply:
What do you think about making it alphabetical by button display name?
I think you can do that before the `join` like this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Reply:
Oh great idea - added!